lib/RTCSession.js

```diff
@@ -377,8 +379,10 @@ module.exports = class RTCSession extends EventEmitter
 
     this._id = this._request.call_id + this._from_tag;
 
-    // Create a new RTCPeerConnection instance.
-    this._createRTCConnection(pcConfig, rtcConstraints);
+    //TODO: AH Emit the MediaConnection?
```

Comment:
Since the user is now providing the media connection, there is no need to fire this event.

Reply:
Ok, i removed the connection emits.